index.html

```diff
@@ -1077,15 +1077,21 @@ <h3>Verifier Caching</h3>
             </p>
          </section>
          <section class="informative">
-            <h3>Content Distribution Networks</h3>
+            <h3>Schema Resolution</h3>
             <p>
                The use of content distribution networks by <a>issuers</a> can increase the
```

Comment:
I would add a sentence at the start here that's generic about schema resolution. Something like...

> Schema resolution is the process of dereferencing a schema's identifier in order to fetch a credential schema.